src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,27 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
+			$args['meta_query'] = array(
+				'relation' => 'OR',
+				array(
+					'key' => '_tribe_aggregator_source_name',
+					'value' => $_REQUEST['s'],
+					'compare' => 'LIKE'
+				),
+				array(
+					'key' => '_tribe_aggregator_import_name',
+					'value' => $_REQUEST['s'],
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 3 space(s) between "'value'" and double arrow, but found 1.
* :warning: Processing form data without nonce verification.
* :warning: Detected usage of a non-sanitized input variable: $_REQUEST['s']

<sub>hash: a0dda2fdb379d67d3f8c896e3b9a9956</sub>